### DISCUSSION TITLE
[A-L] (2023/24) Foglio 6 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#172] [Esercizi - Bertini] [answered]
![Screenshot 2023-11-06 173808](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/db3e3966-fb63-469e-ab65-ea9004583ece)

**Comment by IonutCicio** [comment]
Analogo al punto 4. dell'esercizio #22 _([Foglio 3] Esercizio 5)_

**Comment by Elia-Belli** [comment, marked as answer]
## Soluzione analoga a quella in classe
Come fa notare @CuriousCI il numero di modi di ottenere una k-pla con $n$ figurine distinte è equivalente al numero di funzioni suriettive da $S$ con $|S|=k$ a $S'$ $|S'|=n$. Infatti se $S$ è l'insieme degli "slot" della k-pla e $S'$ l'insieme delle figurine, dobbiamo mappare gli "slot" coprendo tutte le figurine per completare l'album:
$$|\\{f:S\rightarrow S'\ suriettive\\}|=\sum^n_{i=0}(-1)^i\binom{n}{i}(n-i)^k$$
Sempre parlando in termini di funzioni, il numero di tutte le possibili k-ple è il numero di tutte le possibili funzioni sa $S\rightarrow S'$ ovvero $n^k$, detto questo sappiamo che la probabilità è uniforme sulle k-ple, allora:
$$\mathbb{P}(completare\ album)=\frac{1}{n^k}\sum^n_{i=0}(-1)^i\binom{n}{i}(n-i)^k$$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/d16fda69-06e5-474c-9837-a4fc3ba9c110)
